README.md

```diff
@@ -24,12 +24,14 @@ The initial design doc for this work is available [here](doc/design.md).
 
 # Modules
 
-The proxy consists of a core, which is in the [server](server) module, and a set
+The FHIR Info Gateway consists of a core, which is in the [server](server) module, and a set
 of _access-checker_ plugins, which can be implemented by third parties and added
-to the proxy server. Two sample plugins are implemented in the
-[plugins](plugins) module. There is also a sample `exec` module which shows how
-all pieces can be woven together into a single Spring Boot app. To build all
-modules, from the root run:
+to the proxy server. 
+
+* Two sample plugins are implemented in the [plugins](plugins) module. 
```

Comment:
If we want to separate these two in an itemized list, then we should include `server` too, IMO. That is the main part.